nat-lab/tests/test_direct_connection.py

```diff
@@ -451,6 +443,16 @@ async def test_direct_working_paths_with_skip_unresponsive_peers(
     setup_params: List[SetupParameters], _reflexive_ip: str
 ) -> None:
     async with AsyncExitStack() as exit_stack:
+        # Force shorter unresponsive peer handshake threshold
+        # and adjust wireguard keepalives accordingly too
+        # in order to allow for three packet drops
```

Comment:
With threshold being 15, will this be reliable (3*5==15)? Maybe 16s would be safer so that there is some buffer to account for imprecise timers?

Reply:
Well this boundary already accounts for two drops, but yeah, lets target three with a little headroom :) I will change it.

Reply:
:heavy_check_mark: